agent/xds/listeners.go

```diff
@@ -146,19 +146,21 @@ func (s *ResourceGenerator) listenersFromSnapshotConnectProxy(cfgSnap *proxycfg.
 			return nil, err
 		}
 
```

Comment:
It might be helpful to add a small comment around line 135 above (I can't click there in GH) above:
> `filterChain, err := s.makeUpstreamFilterChainForDiscoveryChain`
To imply that the rest of the for-loop after that point is just for tproxy.